src/v1/device_manager_client.ts

```diff
@@ -119,9 +110,24 @@ export class DeviceManagerClient {
    */
   constructor(
     opts?: ClientOptions,
-    gaxInstance?: typeof gax | typeof gax.fallback
+    gaxInstance?: typeof gax | typeof gax.fallback,
+    region?: string,
+    registry?: string,
+    projectId?: string
   ) {
     // Ensure that options include all the required fields.
+    this.region = region;
+    this.registry = registry;
+    this.projectId = projectId;
+    if (region == null || region == '') {
```

Comment:
there are a few lint errors that were introduced in this PR. please fix

Reply:
Sure